docs/2_3_common_states.adoc

```diff
@@ -479,9 +479,9 @@ There are multiple kinds of events that require a transition to <<EventMode>>:
 a|
 * Activate clocks and respective model partitions in latexmath:[\mathbf{f}_{\mathit{event}}]
 * Deactivate clocks and
-  - Reset respective clocked variables to their previous values: latexmath:[\mathbf{v}_{\mathit{d}} := {}^\bullet\mathbf{v}_{\mathit{d}}]
+  - Reset respective clocked variables to their previous values: latexmath:[\mathbf{v}_{\mathit{k}} := {}^\bullet\mathbf{v}_{\mathit{k}}]
   - Deactivate respective model partitions in latexmath:[\mathbf{f}_{\mathit{event}}]
-|<<fmi3SetClock>>
+|<<fmi3SetClockEM>>
```

Comment:
Is the EM intentional?
What does it mean?

Reply:
This is the xref to the fmi3SetClock description for Event Mode (EM).

Reply:
OK thanks